lib/literature/components/forms/publication_form_component.ex

```diff
@@ -99,6 +99,22 @@ defmodule Literature.PublicationFormComponent do
             label="Twitter Description"
           />
         </.accordion>
+        <.accordion id="rss-settings" title="RSS Settings">
+          <.form_field form={f} type="text_input" field={:rss_url} label="RSS URL" />
+          <.form_field form={f} type="text_input" field={:rss_author} label="RSS Author" />
+          <.form_field form={f} type="text_input" field={:rss_email} label="RSS Email" />
+          <div class="space-y-2">
+            <div class="font-semibold text-gray-900">For each post in a feed, include</div>
+            <.radio_buttons
```

Comment:
we have a radio_group component in form_component. You think we can use that? 
https://github.com/martide/literature/blob/80e7b64fb81c8cf07908afa2b42ddee1ca428fc8/lib/literature/components/form_component.ex#L134